contracts/test/v0.8/operatorforwarder/Operator.test.ts

```diff
@@ -299,7 +299,8 @@ describe('Operator', () => {
     })
 
     describe('when called with not enough ETH', () => {
-      it('reverts with subtraction overflow message', async () => {
+      // Test fails with "Uncaught TypeError: Do not know how to serialize a BigInt" for no clear reason
```

Comment:
Test started failing after a version bump, should be fixed when moving these tests to Foundry. No changes will be made to this code without doing a Foundry migration as well.